docs/code/aws/cloud_vm.md

```diff
@@ -1,75 +1,500 @@
 `cloud_vm` deploys MLOps `stack` on top of Cloud provider VMs.
+# ☁️ Cloud Infrastructure Examples
 
+Welcome to the **Cloud Infrastructure** examples! This section contains ready-to-use configurations to help you deploy scalable MLOps stacks on cloud platforms like AWS. Each example is designed to showcase a different tool or workflow, making it easier for you to find and deploy the infrastructure that fits your needs.
 
-## Complete example with all stacks
+## 🚀 Getting Started
+
+### Prerequisites
+Before you begin, ensure you have the following:
+- **Terraform**: Version `>= 1.8.0` installed on your system. [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+- **AWS Account**: Make sure your AWS account is set up and you have appropriate IAM roles.
+- **AWS CLI**: Ensure your AWS CLI is configured with the correct credentials. [Configure AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
+- **Python**: Version `>= 3.7` to run the setup scripts.
 
-=== "Simple Deployment Configuration"
-    ```yaml
-    --8<-- "docs/examples/cloud_vm/complete/aws-complete.yaml"
+### 📦 Installation
+1. **Create a Virtual Environment**:
+    ```bash
+    python -m venv venv
+    source venv/bin/activate  # On Windows use: venv\Scripts\activate
+    ```
+2. **Install `mlinfra`**:
+    ```bash
+    pip install mlinfra
     ```
-=== "Advanced Deployment Configuration"
-    ```yaml
-    --8<-- "docs/examples/cloud_vm/complete/aws-complete-advanced.yaml"
+
+3. **Choose an Example**:
+    Navigate to the example you want to deploy:
+    ```bash
+    cd examples/cloud_infra/dagster
     ```
 
+4. **Configure AWS Credentials**:
+    Modify the `terraform.tfvars` file to update your AWS account details. Make sure your AWS credentials are correctly configured.
```

Comment:
Wait, why are you mentioning this `terraform.tfvars` file? This file is not necessary when you're deploying via mlinfra